lectures/unpleasant.md

```diff
@@ -318,7 +321,7 @@ That leaves the public with less currency but more government interest-bearing b
 
 Since the public has less currency (it's supply has diminished) it is plausible to anticipate that the price level at time $0$ will be driven downward.
 
-But that is not the end of the story, because this ''open market operation'' at time $0$ has consequences for future settings of $m_{t+1}$ and the gross-of-interest government deficit $\bar g_t$. 
+But that is not the end of the story, because this **open market operation** at time $0$ has consequences for future settings of $m_{t+1}$ and the gross-of-interest government deficit $\bar g_t$. 
```

Comment:
`**` is for definitions -- I suggest removing it.

Reply:
Many thanks @jstac, I think open market operation is a definition in this lecture given by 
<img width="850" alt="Screenshot" src="https://github.com/QuantEcon/lecture-python-intro/assets/39026988/879c73bd-a010-4c3b-a77d-2853c658f740">

But I agree it is an edge case and I can remove `**` for all the terms in this screenshot.

Reply:
thanks @jstac I had thought these were definitions as well. So `definitions` should be more direct rather than overall concepts and ideas (as in this case)?

Reply:
Ah, you guys are right.  I didn't see the context.  Thanks for the explanation @HumphreyYang

Reply:
Many thanks @jstac!